projects/aws/upgrader/Makefile

```diff
@@ -34,6 +45,36 @@ PROJECT_DEPENDENCIES=eksd/kubernetes/client eksd/kubernetes/server eksd/cni-plug
 
 include $(BASE_DIRECTORY)/Common.mk
 
+TOOLS_BIN_DIR := $(shell pwd)/hack/tools/bin
+MOCKGEN := $(TOOLS_BIN_DIR)/mockgen
+
+GO=build::common::use_go_version $(GOLANG_VERSION)
```

Comment:
i like this idea, move above the include.  this feels like a pattern we could use more generally for cases where we call go in these makefiles to ensure we are always setting the go_version